features/search.feature

```diff
@@ -214,6 +214,47 @@ Feature: Searching in a journal
         | But I'm better.
         """
 
+    Scenario Outline: Searching by month, day, or year
```

Comment:
This is a great start to the tests. For tests, we want to be sure to include the other journal types (encrypted, directory, day one) and not just the default type.

We've recently found some features that don't work across every journal type, and we didn't even know about it until we expanded testing in this way.

Reply:
Ah! Fair enough. Do you recommend I create new journals for each type based on `dates_similar.journal`, or should I try to adopt the existent journals?

Reply:
I could also modify the existent journals. Thoughts?

Reply:
Done 👍